doc/ug/install_instructions/index.md

```diff
@@ -323,16 +323,16 @@ To get started faster we use the web installer in the following.
 2. Before you proceed ensure that the download didn't get corrupted by verifying the checksum.
 
     ```console
-    $ sha512sum --check Xilinx_Unified_2020.1_0602_1208_Lin64.bin.digests
-   Xilinx_Unified_2020.1_0602_1208_Lin64.bin: OK
+    $ sha512sum --check Xilinx_Unified_{{< tool_version "vivado" >}}_0602_1208_Lin64.bin.digests
+   Xilinx_Unified_{{< tool_version "vivado" >}}_0602_1208_Lin64.bin: OK
```

Comment:
I'm not sure whether that's right: what do the `0602_1208` parts come from? Are they the same across versions? Similarly with the next change.

Reply:
Yes, I agree that this is not a clean way to do this. These parts in the file name are very likely not the same across versions. As suggested by @imphil I am going to revert the changes on the install instructions to make them match the screenshots.